src/Datadog.Trace/Propagation/B3/B3HttpHeaderNames.cs

```diff
@@ -0,0 +1,30 @@
+namespace Datadog.Trace.Propagation.B3
```

Comment:
Personally, I think that there is no need of `B3` folder and namespace. `B3` prefixes is names is enough

Reply:
Agree, I first thought there will be more files there. But seems the final result was just 2, so flattened out both implementations.